src/js/components/Select/__tests__/Select-test.js

```diff
@@ -326,4 +326,21 @@ describe('Select', () => {
       expect(document.activeElement).toMatchSnapshot();
     });
   });
+
+  test('empty results search', () => {
```

Comment:
Thanks for taking the time to add a test. 💯 
It will be useful to have a quick showcase/test on select.stories.js as well. either Select->Search or Select->Custom Search can have a simple line of emptySearchMessage="No matches found" on the Select component (e.g. select.stories.js line 419) that will demonstrate this option.

Reply:
sure done :)